Rick-and-Morty/Rick And Morty/Views/CharacterCard.swift

```diff
@@ -63,32 +43,64 @@ struct CharacterCard: View {
             RoundedRectangle(cornerRadius: constants.cornerRadius)
                 .stroke(Color(.lightGray), lineWidth: constants.borderWidth)
         )
-        .padding()
         .frame(height: constants.cardHeight)
     }
+}
+
+extension CharacterCard {
+    private struct DescriptionDetailView: View {
+        let title: String
+        let text: String
+        
+        var body: some View {
+            VStack(alignment: .leading, spacing: 0) {
+                Text(title)
+                    .font(.caption)
+                    .foregroundColor(.secondary)
+                Text(text)
+            }
+        }
+    }
     
-    func characterDetailsVStack(caption: String, text: String) -> some View {
-        return VStack(alignment: .leading, spacing: constants.noSpacing) {
-                        Text(caption)
-                            .font(.caption)
-                            .foregroundColor(.secondary)
-                        Text(text)
+    private struct NameAndStatusView: View {
+        let name: String
+        let statusColor: Color
+        let statusText: String
+        
+        var body: some View {
+            VStack(alignment: .leading, spacing: 0) {
+                Text(name)
+                    .font(.title)
+                    .fontWeight(.black)
+                HStack {
+                    Circle()
+                        .foregroundColor(statusColor)
+                        .frame(width: 10, height: 10)
+                    Text(statusText)
+                }
+            }
         }
     }
 }
 
+
+
+
+
+
 struct CharacterTileView_Previews: PreviewProvider {
     static var previews: some View {
         Group {
             VStack {
-                CharacterCard(characterCardState: CharacterCardState(id: 2, name: "Morty", image: UIImage(named: "morty-image")!, isAlive: true, species: "Human", lastLocation: "Earth", firstEpisode: "Episode 1"))
+                CharacterCard(cardViewModel: CharacterCardViewModel(character: Character(id: 1, name: "Morty", species: "Human", lastLocation: LastLocation(name: "Earth", url: ""), status: .alive, imageURL: "", episodeURLs: [])))
```

Comment:
I like that you're using previews. Have you thought of all the possible display views?